src/applications/vaos/tests/list/va-appointments.unit.spec.jsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import { expect } from 'chai';
+import moment from 'moment';
+import set from 'platform/utilities/data/set';
+import { renderInReduxProvider } from 'platform/testing/unit/react-testing-library-helpers';
+import reducers from '../../reducers';
+import singleVAAppointment from '../mocks/single-va-appointment.json';
+import singleVAFacility from '../mocks/single-va-facility.json';
+import { mockAppointmentInfo, mockFacilitesFetch } from '../mocks/helpers';
+
+import FutureAppointmentsList from '../../components/FutureAppointmentsList';
+
+const initialState = {
+  featureToggles: {
+    vaOnlineSchedulingCancel: true,
+  },
+};
+
+describe('VAOS appointment list future booked', () => {
+  describe('va appointments', () => {
+    it('should show information without facility details', async () => {
+      const appointment = set(
+        'attributes.startDate',
+        moment()
+          .add(3, 'days')
+          .format(),
+        singleVAAppointment,
+      );
+      mockAppointmentInfo({ va: [appointment] });
+      const { findByText, baseElement, getByText } = renderInReduxProvider(
+        <FutureAppointmentsList />,
```

Comment:
We could in theory create a helper to render everything starting from routes, but I'm not sure that's worth the effort, especially if Cypress is easier to deal with than Nightwatch.